apps/autoscheduler/test_function_dag.cpp

```diff
@@ -81,6 +78,102 @@ int main(int argc, char **argv) {
 
     // Disabled for now: there is still work to do to populate the jacobian
     //assert(with_extern.str() == without_extern.str());
+}
+
+extern "C" int matmul(
+    halide_buffer_t *input1,
+    halide_buffer_t *input2,
+    halide_buffer_t *output) {
+
```

Comment:
Nit: first line of function should not be blank

Reply:
done